hack/install.sh

```diff
@@ -19,11 +21,38 @@ set -e
 #
 
 url='https://get.docker.com/'
+wait_if_docker_command_exists=1
+reinstall_if_docker_command_exists=1
 
 command_exists() {
 	command -v "$@" > /dev/null 2>&1
 }
 
+show_help() {
+	cat <<-EOF
+	Usage: install.sh  [-w true/false] [-r true/false] -h
+	  -w : if docker command already exists, wait for a few seconds before continuing. Default is true.
+	  -r : if docker command already exists, reinstall it. Mark it false to skip and exit successfully. Default is true. 
+	  -h : display this help
+	EOF
+}
+
+parse_arguments(){
+        while getopts ":hw:r:" opt; do
+                case "$opt" in
+                        h)  show_help
+                            exit 0
+                            ;;
+                        w)  if  [ "$OPTARG" == "false" ]; then wait_if_docker_command_exists=0 ; fi
```

Comment:
Look. I don't like this IF. But anything else I've tried ended up in the 0-or-1 is success trap, which just made the code insane to follow.